imbi/endpoints/project_ssm_configuration.py

```diff
@@ -108,19 +108,27 @@ async def get(self, *args, **kwargs) -> None:
         self.send_response(output)
 
     async def post(self, *args, **kwargs) -> None:
+        project_info = await self._get_project_info(kwargs['project_id'])
         google_tokens = await self._get_google_tokens()
         aws_session = aioboto3.Session()
         body = self.get_request_body()
-        for param in body['values']:
+
+        ssm_path_prefix = self.path_prefix(
+            self.application.settings['project_configuration']
+            ['ssm_prefix_template'], project_info['project_slug'],
+            project_info['project_type_slug'], project_info['namespace_slug'])
```

Comment:
This is one of those places where I would use a temporary var just to make the code less scrambled.

```suggestion
        project_config = self.application.settings['project_configuration']
        ssm_path_prefix = self.path_prefix(
            project_config['ssm_prefix_template'],
            project_info['project_slug'], project_info['project_type_slug'],
            project_info['namespace_slug'])
```